docs/src/test/scala/docs/http/scaladsl/server/FullSpecs2TestKitExampleSpec.scala

```diff
@@ -53,8 +53,8 @@ class FullSpecs2TestKitExampleSpec extends Specification with Specs2RouteTest {
     "return a MethodNotAllowed error for PUT requests to the root path" in {
       // tests:
       Put() ~> Route.seal(smallRoute) ~> check {
-        status === StatusCodes.MethodNotAllowed
-        responseAs[String] === "HTTP method not allowed, supported methods: GET"
+        status shouldEqual StatusCodes.MethodNotAllowed
```

Comment:
Maybe let's keep it consistent by keeping `shouldEqual` even if we lose a bit of typesafety here right now. Could make sense to follow up on *all* example using the typesafe variant in the future. WDYT, @jonas?